pkgs/gnolang/gnomod/gnomod.go

```diff
@@ -102,3 +108,12 @@ func GnoToGoMod(f File) (*File, error) {
 
 	return &f, nil
 }
+
+func isReplaced(module module.Version, repl []*modfile.Replace) (*module.Version, bool) {
```

Comment:
Why do you need to return a reference to the `module.Version`, since it doesn't seem to require it?

https://pkg.go.dev/golang.org/x/mod/module#pkg-overview
```go
type Version struct {
	Path string
	Version string
}
```

Why not just return the struct itself? That way you don't have to dereference it when you use `isReplaced`

Reply:
Although I don't have a problem with your suggestion too.

Considering dereferencing is cheap. I prefer returning `nil` instead of an empty struct.